dev/lint-python

```diff
@@ -22,6 +22,7 @@ SPARK_ROOT_DIR="$(dirname "$SCRIPT_DIR")"
 PATHS_TO_CHECK="./python/pyspark/ ./ec2/spark_ec2.py ./examples/src/main/python/ ./dev/sparktestsupport"
 PATHS_TO_CHECK="$PATHS_TO_CHECK ./dev/run-tests.py ./python/run-tests.py"
 PYTHON_LINT_REPORT_PATH="$SPARK_ROOT_DIR/dev/python-lint-report.txt"
```

Comment:
`PYTHON_LINT_REPORT_PATH` and `PYLINT_REPORT_PATH` are confusingly-similar variable names.  Can we rename the first variable to `PEP8_REPORT_PATH` to help disambiguate?